tox.ini

```diff
@@ -105,6 +105,8 @@ whitelist_externals =
     {dev,tests,functests,pip-compile}: sh
 changedir =
     {docs,checkdocs}: docs
+depends =
+    {coverage,functests}: tests
```

Comment:
This informs tox to wait until the tests env has finished running before starting either the coverage or functests envs, whenever it's asked to run those envs in a single command. Functests has to run after tests because they both use the same Pg DB and Es index, so they can't run at the same time.